app/controllers/comments_controller.rb

```diff
@@ -2,6 +2,8 @@ class CommentsController < ApplicationController
   before_action :set_comment, only: %i[ show edit update destroy ]
   before_action :is_an_authorized_user, only: [:destroy, :create]
```

Comment:
redundant due to before_action { authorize @comment || Comment}

Reply:
Removed. Thanks.